pkgs/oauth2/README.md

```diff
@@ -161,6 +162,46 @@ For Flutter apps, there's two popular approaches:
       );
     ```
 
+
+1.  To handle redirect on Flutter Web you would need to add a html file to the web folder with some 
+additional JS code to handle the redirect back to the app (in this example the code will be saved 
+and passed through localStorage).
```

Comment:
This kind of depends on how you configure your oauth2 flow and what you're redirect URL is, doesn't it?

Reply:
While he redirect behavior on Flutter Web does depend on how the OAuth2 flow is configured and the redirect URI set in the provider - the example I added is meant to provide a minimal working solution for handling redirects specifically when using implicit or authorization code flows with a custom redirect HTML page (which I think is the most common use case).

The intent is to give developers a concrete starting point, since the existing documentation and issues (e.g., #350) don’t cover a practical approach for Web, and a lot of stackoverflow questions/issues indicate that developers are often confused by this.

Reply:
I guess this is good enough as a starting point.